src/Bicep.Core/Syntax/ExplicitVariableAccessSyntax.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.Core.Syntax
+{
+    /// <summary>
+    /// Represents an explicit reference to a variable
+    /// </summary>
+    public class ExplicitVariableAccessSyntax: VariableAccessSyntax
```

Comment:
>ExplicitVariableAccessSyntax [](http://example.com/codeflow?start=17&length=28)

I do wonder if we should move this to an Emit-related namespace since this type of syntax node can't ever be produced by the parser.